### DISCUSSION TITLE
So, which opengrok release I should stick on if only option for tomcat is 9.0.64?

### DISCUSSION THREAD

**Discussion by nikkej** [oracle/opengrok#4011] [Q&A]
I'm running fedora f35 on my desktop and only option for tomcat seems to be 9.0.64. Well, I tried opengrok 1.7.34 on tomcat 9 but as expected, not so good result (indexer works but it is source.war causing headaches). Have to mention that I have installed and run opengrok 0.12.1.6 years successfully

**Comment by vladak** [comment]
Yep, Tomcat 10 is minimum version. There is OpenGrok Docker image.
